### DISCUSSION TITLE
How to disable custom rule for authenticated users?

### DISCUSSION THREAD

**Discussion by mvasi90** [rspamd/rspamd#4810] [Q&A] [answered]
I want to replace the subject for e-mail that contains unmatched links (phishing) and possible spoofing.

To do that I created a custom rules in `local.d/force_actions.conf`

```
rules {
  PHISHING {
    expression = "PHISHING"
    action = "rewrite subject"
    subject = "[Enlace falsificado: PHISHING] %s"
    honor_action = ["reject", "soft reject", "greylist"]
  }

  SPOOFING {
    expression = "VIOLATED_DIRECT_SPF | DMARC_POLICY_REJECT | DMARC_POLICY_QUARANTINE"
    action = "rewrite subject"
    subject = "[Suplantación de identidad] %s"
    honor_action = ["reject"]
  }
}
```

I discovered that all the outgoing e-mails (authenticated users) are filtered by rspamd. This is fine to prevent spam on a compromised system/account. But I want to disable my custom rules from authenticated users. I can do that?

**Comment by fatalbanana** [comment, marked as answer]
Symbols can be conditionally disabled using [settings module](https://rspamd.com/doc/configuration/settings.html).

~~~hcl
# local.d/settings.conf
authed_users {
  authenticated = yes;
  apply {
    groups_disabled = ["force_actions"]; # this group exists as of Rspamd 3.8.x
    # symbols_disabled = ["FORCE_ACTION_PHISHING", "FORCE_ACTION_SPOOFING"]; # pre Rspamd 3.8.x
  }
}
~~~

**Comment by mvasi90** [comment]
It works. Thank you.

But I would like to have more control. Instead disabling all the actions, would be nice to disable just a custom rule or rules.

By the way, why this option is not documented?
`authed_users` not appears on `user settings`.

**Comment by fatalbanana** [comment]
> But I would like to have more control. Instead disabling all the actions, would be nice to disable just a custom rule or rules.

See `symbols_disabled`

> By the way, why this option is not documented? `authed_users` not appears on `user settings`.

This is a name for the group, it can be anything
